x-pack/plugins/security_solution/public/management/hooks/use_with_show_responder.tsx

```diff
@@ -81,18 +82,21 @@ export const useWithShowResponder = (): ShowResponseActionsConsole => {
           'data-test-subj': `${agentType}ResponseActionsConsole`,
           storagePrefix: 'xpack.securitySolution.Responder',
           TitleComponent: () => {
-            if (agentType === 'endpoint') {
-              return <HeaderEndpointInfo endpointId={agentId} />;
-            }
-            if (agentType === 'sentinel_one') {
+            if (agentStatusClientEnabled || agentType === 'sentinel_one') {
```

Comment:
For future usages: maybe worth to change it to: `if (agentStatusClientEnabled || agentType !== 'endpoint')`?

Reply:
yeah for some reason my editor didn't like that earlier. But looks like it seems to not complain. Changed it to be more generic.